service/worker/workerdeployment/util.go

```diff
@@ -60,24 +60,22 @@ const (
 	DeleteDeployment             = "delete-deployment"                // for WorkerDeployment wfs
 
 	// Signals
-	ForceCANSignalName      = "force-continue-as-new" // for Worker Deployment Version _and_ Worker Deployment wfs
-	SyncDrainageSignalName  = "sync-drainage-status"
-	TerminateDrainageSignal = "terminate-drainage"
+	ForceCANSignalName       = "force-continue-as-new" // for Worker Deployment Version _and_ Worker Deployment wfs
+	SyncDrainageSignalName   = "sync-drainage-status"
```

Comment:
```suggestion
	SyncDrainageSignalName   = "sync-drainage-status" // for Worker Deployment Version _and_ Worker Deployment wfs
```

Reply:
I'm not using SyncDrainageSignalName in the deployment wf